lib/Models/CswCatalogGroup.ts

```diff
@@ -325,7 +325,10 @@ class CswStratum extends LoadableStratum(CswCatalogGroupTraits) {
       records.push(...(json?.SearchResults?.Record ?? []));
 
       // Get next start position - or stop pageing
-      const nextRecord = json?.SearchResults?.nextRecord;
+      const nextRecord =
+        typeof json?.SearchResults?.nextRecord === "string"
+          ? parseInt(json?.SearchResults?.nextRecord ?? "0")
+          : json?.SearchResults?.nextRecord;
```

Comment:
When paging for CSW records, we use the `nextRecord` ID in the response to set `startPosition`. 

This was a `string`, but should have been a `number` - so we were getting this error:

```
startPosition must be an integer greater than zero.
```